kernel/environ.mli

```diff
@@ -133,6 +133,7 @@ val lookup_rel    : int -> env -> Constr.rel_declaration
 val lookup_rel_val : int -> env -> lazy_val
 val evaluable_rel : int -> env -> bool
 val env_of_rel     : int -> env -> env
+val ids_of_rel_context : rel_context -> Id.Set.t
```

Comment:
This is not the right place for this function. Either put it in Context or simply inline it.

Reply:
It was by symmetry with `Environ.ids_of_named_context_val` but I agree that it is worth moving it to context.ml which already has the equivalent `Named.to_vars`.